scripts/get-workspace-map.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# get-workspace-map.sh context-1 context-2... context-n
+#   Read Namespace resources and generate a JSON object mapping from namespaces
+#   to workspace based on the value of label toolchain.dev.openshift.com/space.
+#   This script assumes `oc` is preconfigured with all the required clusters
+#   and could be found in $PATH
+#
+set -o pipefail -o errexit -o nounset
+
+out={}
+for context in "$@"
+do
+  oc config use-context "$context"
```

Comment:
This is prone to race conditions, perhaps we can use the `--context` parameter instead? Or maybe there is some env var we can use?

Reply:
Can't we do something like `oc get namespaces --context="$context"`

Reply:
I'll give it a try. I was not aware of that option.

Reply:
done